CodeEditCli/Sources/CodeEditCli/main.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  CodeEditCli.swift
+//  
+//
+//  Created by Ben Koska on 14.06.22.
+//
+
+import Foundation
+
+func convertToAbsolutePath(_ path: String) -> String {
+    let nsString = NSString(string: path)
+    if nsString.isAbsolutePath {
+        return nsString.standardizingPath
+    }
+
+    return String(
+        URL(
+            string: path,
+            relativeTo: URL(
+                fileURLWithPath: FileManager.default.currentDirectoryPath
+            )
+        )?.pathComponents.joined(separator: "/").dropFirst(1) ?? ""
+    )
+}
+
+func openApp(paths: [String]? = nil) {
+    let task = Process()
+    task.launchPath = "/usr/bin/open" // This should be the same on all installations of MacOS
+
+    task.arguments = ["-a", "CodeEdit"]
+
+    if let paths = paths {
+        task.arguments?.append("--args")
+        for path in paths {
+            task.arguments?.append("--open")
+            task.arguments?.append(convertToAbsolutePath(path))
+        }
+    }
+
+    task.launch()
```

Comment:
Why not use the bundle identifier?
What if someone renames the app to `Code Editor.app` for example, will this command still work?

IIRC you can use it like
```swift
        guard let codeEdit = NSWorkspace.shared.urlForApplication(
            withBundleIdentifier: "austincondiff.CodeEdit"
        ) else { return }

        NSWorkspace.shared.open(
            [convertToAbsolutePath(path)],
            withApplicationAt: codeEdit,
            configuration: NSWorkspace.OpenConfiguration()
        )
```
This should work.

See https://developer.apple.com/documentation/appkit/nsworkspace/3172702-open for more information